src/Pelagos/Bundle/AppBundle/Resources/views/Email/data-repository-managers.error-remotely-hosted.email.twig

```diff
@@ -0,0 +1,14 @@
+{% block subject 'Error Log - List of Remotely Hosted Datasets links failed '%}
+
+{% block body_html %}
+    <h3>
+        Below are the list of Dataset Udi's which have failed.
+    </h3>
+
+    <br>
+
+    {% for udi in listOfUdi %}
+        <li>{{ udi }}</li>
```

Comment:
Do we also want to give them a reason, or error code? Or maybe even the link?

Reply:
We can push it to the PO, and if they want changes to the email sent, we can add in more stuff